src/index.js

```diff
@@ -1,47 +1,64 @@
-import Mario from './arts/mario.ansi';
```

Comment:
Unfortunately not, `cp` doesn't support such kind of syntax when you copy all contents of a directory.